Cargo.toml

```diff
@@ -18,7 +18,7 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 # crates.io
 futures         = { version = "0.3" }
-parity-util-mem = { version = "0.9", default-features = false, features = ["jemalloc-global"] }
+parity-util-mem = { version = "0.10", default-features = false, features = ["jemalloc-global"] }
```

Comment:
Do we need this explicit dependency? I did not find usage anywhere in this repo or darwinia-common.